nemo/lightning/_strategy_lib.py

```diff
@@ -399,7 +399,7 @@ def enable_nvidia_optimizations() -> None:
 
 
 def optimizer_sharded_state_dict(
-    model: SharedStateDictProtocol, optimizer: "Optimizable", is_loading=False
+    model: SharedStateDictProtocol, optimizer: "Optimizable", is_loading=False, sharding_type='dp_zero_gather_scatter'
```

Comment:
One last comment, can you make `fully_sharded_model_space` a default sharding type? Thanks
```suggestion
    model: SharedStateDictProtocol, optimizer: "Optimizable", is_loading=False, sharding_type='fully_sharded_model_space'
```